hub/powertoys/screen-ruler.md

```diff
@@ -19,9 +19,9 @@ Press <kbd>⊞ Win</kbd>+<kbd>Shift</kbd>+<kbd>M</kbd> to activate and then sele
 ## How to use
 
 - Bounds (Dashed square symbol): This is a bounding box. Click and drag with your mouse. If you hold <kbd>Shift</kbd>, the box(es) will stay in place until you cancel the interaction.
-- Spacing (&#9547;): This will measure horizontal and vertical spacing at the same time.  Click the symbol and move your mouse to your target location.
-- Horizontal (&#9473;): This will measure only horizontal spacing. Click the symbol and move your mouse pointer to your target location.
-- Vertical (&#9475;): This will measure only vertical spacing. Click the symbol and move your mouse pointer to your target location.
+- Spacing (↔<span style="position:relative; right:9px; top:-1px">↕</span><span style="position:relative; right:7px">): This will measure horizontal and vertical spacing at the same time.  Click the symbol and move your mouse to your target location.</span>
```

Comment:
Is there another way to achieve this change. The current HTML is causing a new build warning.

Reply:
There is no other way to render this icon. We could just leave it out.

Reply:
OK. Let's try to push this through. I had one warning that prevented the PR from syncing to the live site. If that happens, I'll follow up with another PR to remove the icon. Does this need to wait for the release this week?